README.md

```diff
@@ -436,21 +455,46 @@ map<json> ordersMap;
 service<http:Service> orderMgt bind listener {
 ``` 
 
-- Here we have used `@kubernetes:Deployment` to specify the docker image name which will be created as part of building this service. 
+- Here we have used `@kubernetes:Deployment` to specify the Docker image name which will be created as part of building this service.
```

Comment:
Minor corrections:
Here, 
"`@kubernetes:Deployment` is used to specify the Docker image name that should be created as part of building the service.
`@kubernetes:Service` is specified to creates a Kubernetes service that exposes the Ballerina service that is running on a Pod.
`@kubernetes:Ingress` is used as the external interface to access your service (with path `/` and host name `ballerina.guides.io`).
Now, use the following command to build a Ballerina executable archive (.balx) of the service that you developed above. This creates the corresponding Docker image and the Kubernetes artifacts using the Kubernetes annotations that you have configured."